spec/rspec/core/configuration_spec.rb

```diff
@@ -1229,7 +1229,171 @@ def metadata_hash(*args)
       end
     end
 
+    describe "color options" do
+      before { config.no_color = false }
+
+      context "#force_color=false, #no_color=false" do
+        context "with output.tty?" do
+          it "sets color_enabled?" do
+            output = StringIO.new
+            config.output_stream = output
+
+            allow(config.output_stream).to receive_messages :tty? => true
```

Comment:
Not a merge blocker, but every where else our style for this kind of thing is to use parens for the `recieve_messages` call:

``` ruby
allow(config.output_stream).to receive_messages(:tty? => true)
```

It'd be nice for consistency reasons if you could update this.